rustls/src/server/server_conn.rs

```diff
@@ -242,6 +242,30 @@ impl<'a> ClientHello<'a> {
 /// * [`ServerConfig::cert_compression_cache`]: caches the most recently used 4 compressions
 /// * [`ServerConfig::cert_decompressors`]: depends on the crate features, see [`compress::default_cert_decompressors()`].
 ///
+/// # Sharing resumption storage between `ServerConfig`s
+/// In a program using many `ServerConfig`s it may improve resumption rates
+/// (which has a significant impact on connection performance) if those
+/// configs share [`ServerConfig::session_storage`] or [`ServerConfig::ticketer`].
+///
+/// However, caution is needed: other fields influence the security of a session
+/// and resumption between them can be surprising.  If sharing
+/// [`ServerConfig::session_storage`] or [`ServerConfig::ticketer`] between two
+/// `ServerConfig`s, you should also evaluate the following fields and also
+/// ensure they are equivalent:
+///
+/// * `ServerConfig::verifier` -- client authentication requirements,
```

Comment:
should this field be in `[]`'s to match the `cert_resolver` instance below?

Reply:
Unfortunately this is a private field, so asking to link it causes a warning.